.markdownlint.json

```diff
@@ -0,0 +1,56 @@
+{
+   "default": true,
+   "header-increment": true,
+   "heading-style": {
+      "style": "atx"
+   },
+   "ul-style": {
+      "style": "asterisk"
+   },
+   "list-indent": true,
+   "ul-start-left": false,
+   "ul-indent": {
+      "indent": 3,
+      "start_indented": true
+   },
+   "no-trailing-spaces": true,
+   "no-hard-tabs": true,
+   "no-reversed-links": true,
+   "no-multiple-blanks": { "maximum": 2 },
+   "commands-show-output": true,
+   "line-length": {
+      "line_length": 90,
+      "code_block_line_length": 130
+   },
+   "no-missing-space-atx": true,
+   "no-multiple-space-atx": true,
+   "heading-start-left": true,
+   "no-duplicate-header": {
+      "allow_different_nesting": true
+   },
+   "no-trailing-punctuation":{
+      "punctuation":  ".,;:"
+   },
+   "no-bare-urls": true,
+   "hr-style": { "style": "---" },
+   "no-emphasis-as-heading": false,
+   "no-space-in-emphasis": true,
+   "no-space-in-code": true,
+   "no-space-in-links": true,
+   "fenced-code-language": true,
+   "first-line-heading": false,
+   "no-empty-links": false,
+   "proper-names": {
+      "names": [
+         "JavaScript",
+         "Vue.js",
+         "TypeScript",
+         "HTML",
+         "jQuery"
```

Comment:
Let's add:

 * `Silvermine` since it sometimes slips in with "M"
 * `ESLint`

@yokuze @jimjenkins5 can you think of any other common proper names we use that have odd capitalization to catch here?

Reply:
* `YML`
* `YAML`
* `URL`
* `Node.js`
* `SCSS`
* `CSS`

?

Reply:
👍

Reply:
It would be nice if we could also test to make sure that markdownlint doesn't make us capitalize them if they appear in a filename. e.g. "languages.yml" vs "languages.YML"

Reply:
We can disable the rule for filenames inside backticks via the [`code_blocks` parameter](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md044---proper-names-should-have-the-correct-capitalization), but it's still triggered for links in statements like this:

```
See [@silvermine/eslint-config][eslintconfig] and
[@silvermine/eslint-plugin][eslintplugin] for those standards.
```

We could work around it by placing all links with enforced names in backticks like so:

```
See [`@silvermine/eslint-config`][eslintconfig] and
[`@silvermine/eslint-plugin`][eslintplugin] for those standards.
```

But that's not very intuitive. I recommend disabling the rule for now while I open a PR to ignore enforced proper names in links. Given my experience with the project, it should only take me ~8,000 hours give or take.